distribution/docs/src/main/resources/content/_quickstart/quickstart-ingesting.adoc

```diff
@@ -26,11 +26,11 @@ XML metadata for text searching is not automatically generated from GeoJson fiel
 
 Querying from the Search UI (${secure_url}/search) will return the record for the file ingested:
 
-. Select the Menu icon (image:navigator-menu-icon.png[])and return to *Workspaces*.
+. Select the Menu icon (image:navigator-icon.png[navigator icon])and return to *Workspaces*.
 . Select the example workspace *New Workspace*.
-. Select *Add Search*
-. Enter `*` in the Text search box and select *Save*.
-. Select the *Options* icon (options-icon.png[]) and select *Run* to run query.
+. Select *Add Search*.
+. Enter `*` in the Text search box and select **Save**.
+. Select the *Options* icon (image:options-icon.png[options icon]) and select *Run* to run query.
```

Comment:
❓ Isn't there a 'google' search box now? I don't think it's necessary to create a workspace first.

Reply:
fixed.